skrebate/relieff.py

```diff
@@ -7,28 +7,115 @@
     - Ryan J. Urbanowicz (ryanurb@upenn.edu)
```

Comment:
Most changes here are for flake8. I didn't flake8 other files that I didn't work on